### PR TITLE
[WIP] Final update PR of Cluster Serving for 0.7.0

### PR DESCRIPTION
**Pull request by Litchilitchy**
updated readme, about to update docker related.

### CODE REVIEW

File: docker/cluster-serving/docker/DockerFile [hunk 1]
```diff
@@ -0,0 +1,65 @@
+FROM ubuntu:18.04
```
Comment (glorysdj) [inline docker/cluster-serving/docker/DockerFile]: docker/cluster-serving/DockerFile ？

File: docker/cluster-serving/config.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
+## Analytics-zoo Cluster Serving
+
+model:
+  # model path must be set
+  path: /opt/work/model/
+data:
```
Comment (glorysdj) [inline docker/cluster-serving/config.yaml]: data source?

File: docker/cluster-serving/config.yaml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  # default, localhost:6379
+  src: localhost:6379
```
Comment (glorysdj) [inline docker/cluster-serving/config.yaml]: redis url?

File: docker/cluster-serving/docker/DockerFile [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+#redis
+RUN wget http://download.redis.io/releases/redis-5.0.5.tar.gz && \
+    tar xzf redis-5.0.5.tar.gz && \
+    rm redis-5.0.5.tar.gz && \
+    cd redis-5.0.5 && \
+    make && \
+    cd ..
+
+ARG ANALYTICS_ZOO_VERSION=0.5.1
```
Comment (glorysdj) [inline docker/cluster-serving/docker/DockerFile]: rm?

File: docker/cluster-serving/docker/DockerFile [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+ARG BIGDL_VERSION=0.8.0
```
Comment (glorysdj) [inline docker/cluster-serving/docker/DockerFile]: 0.10.0?

File: docker/cluster-serving/docker/DockerFile [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+ENV REDIS_HOST			loaclhost
+ENV REDIS_PORT			6379
+
+ENV SPARK_HOME			/opt/work/spark-${SPARK_VERSION}
+ENV ANALYTICS_ZOO_HOME		/opt/work/analytics-zoo-${ANALYTICS_ZOO_VERSION}/dist
+ENV JAVA_HOME			/opt/jdk
+ENV CONDA_HOME			/opt/work/conda
+ENV PATH			${JAVA_HOME}/bin:${CONDA_HOME}/bin:${PATH}
+    
+ADD ./docker docker
```
Comment (glorysdj) [inline docker/cluster-serving/docker/DockerFile]: should download analytics zoo jar from public nexus

File: docker/cluster-serving/docker/DockerFile [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+ENV SPARK_OPTS			"--master local[*]"
+ENV REDIS_HOST			loaclhost
+ENV REDIS_PORT			6379
+
+ENV SPARK_HOME			/opt/work/spark-${SPARK_VERSION}
+ENV ANALYTICS_ZOO_HOME		/opt/work/analytics-zoo-${ANALYTICS_ZOO_VERSION}/dist
+ENV JAVA_HOME			/opt/jdk
+ENV CONDA_HOME			/opt/work/conda
+ENV PATH			${JAVA_HOME}/bin:${CONDA_HOME}/bin:${PATH}
+
```
Comment (glorysdj) [inline docker/cluster-serving/docker/DockerFile]: rm unnecessary envs

Comment (glorysdj) [reply docker/cluster-serving/docker/DockerFile]: also format them

File: docker/cluster-serving/docker/DockerFile [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+ENV SPARK_HOME			/opt/work/spark-${SPARK_VERSION}
+ENV ANALYTICS_ZOO_HOME		/opt/work/analytics-zoo-${ANALYTICS_ZOO_VERSION}/dist
+ENV JAVA_HOME			/opt/jdk
+ENV CONDA_HOME			/opt/work/conda
+ENV PATH			${JAVA_HOME}/bin:${CONDA_HOME}/bin:${PATH}
+    
+ADD ./docker docker
+
+ADD ./zoo-cluster-serving.sh /opt/work
```
Comment (glorysdj) [inline docker/cluster-serving/docker/DockerFile]: should wget this from github repo

**Comment by Litchilitchy** [comment]
Jenkins passed @glorysdj

File: docker/cluster-serving/docker-build.sh [hunk 9]
```diff
@@ -0,0 +1,6 @@
+docker build \
+    --build-arg http_proxy=http://10.239.4.101:913/ \
```
Comment (glorysdj) [inline docker/cluster-serving/docker-build.sh]: change to myproxyhost:myproxyport

File: docker/cluster-serving/DockerFile [hunk 10]
```diff
@@ -0,0 +1,59 @@
...
+    rm spark-${SPARK_VERSION}-bin-hadoop2.7.tgz
+
+
+ENV SPARK_HOME			/opt/work/spark-${SPARK_VERSION}
+ENV ANALYTICS_ZOO_HOME		/opt/work/analytics-zoo-${ANALYTICS_ZOO_VERSION}/dist
+ENV JAVA_HOME			/opt/jdk
+
+ENV PATH			${JAVA_HOME}/bin:${CONDA_HOME}/bin:${PATH}
+
+RUN wget https://oss.sonatype.org/content/repositories/public/com/redislabs/spark-redis/2.4.0/spark-redis-2.4.0-jar-with-dependencies.jar
```
Comment (glorysdj) [inline docker/cluster-serving/DockerFile]: 2.4.0 change to ${SPARK_VERSION}?

File: docker/cluster-serving/DockerFile [hunk 11]
```diff
@@ -0,0 +1,59 @@
...
+
+RUN wget https://oss.sonatype.org/content/repositories/snapshots/com/intel/analytics/zoo/
```
Comment (glorysdj) [inline docker/cluster-serving/DockerFile]: why not get a jar?

**Pull request merged**